tests/data/serialization/ndjson/test_conversational_text.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+import labelbox.types as lb_types
+from labelbox.data.serialization import NDJsonConverter
+
+radio_ndjson = [{
```

Comment:
Any reason why a new file was created for this instead of adding to `test_conversation.py`?